tee/Cargo.toml

```diff
@@ -21,6 +21,7 @@ log = "0.4.27"
 sea-orm = "1.1.11"
 serde = "1.0.219"
 serde_json = "1.0.140"
+shielder-prover-common = { path = "crates/shielder-prover-common" }
```

Comment:
Not related to this particular line, but I am wondering if we should have different repository structure, ie we have `zkOS-monorepo/crates` and it is there when we should put all Rust crates. This is because we want to benefit from the common `Cargo.lock` file, and also have reproducible builds kind-of (use the same dependencies in every production binary). So the alternative idea is e.g. to have
```
zkOS-monorepo/
   crates/
       shielder-prover-server-host/
       shielder-prover-server-tee/
       shielder-prover-server-common/
```
and then leave `zkOS-monorepo/tee` folder with current contents, that's a PoC solution for different project. In this way, we  know that what's in `crates/` is production-ready code. Wdyt?

Reply:
Superb, makes sense, created https://cardinal-cryptography.atlassian.net/browse/BAC-186